packages/components/addon/components/hds/form/toggle/field.hbs

```diff
@@ -1,18 +1,13 @@
-<div class={{this.classNames}}>
-  <Hds::Form::Toggle::Base
-    {{! NOTICE: we need to use this special argument to pass a class to the wrapping element, because the ...attributes is applied to the control }}
-    {{! template-lint-disable no-capital-arguments }}
-    @_wrapperClass="hds-form-field__control"
-    @isInvalid={{@isInvalid}}
-    ...attributes
-    id={{this.id}}
-    aria-describedby={{this.ariaDescribedBy}}
-  />
-  {{yield (hash Label=(component "hds/form/label" controlId=this.id _contextualClass="hds-form-field__label"))}}
-  {{yield
-    (hash
-      HelperText=(component "hds/form/helper-text" controlId=this.id _contextualClass="hds-form-field__helper-text")
-    )
-  }}
-  {{yield (hash Error=(component "hds/form/error" controlId=this.id _contextualClass="hds-form-field__error"))}}
-</div>
\ No newline at end of file
+<Hds::Form::Field @isInvalid={{@isInvalid}} @layout="flag" as |F|>
```

Comment:
[chore] can you double check that this is still needed?
```suggestion
<Hds::Form::Field @layout="flag" as |F|>
```

Reply:
Not needed, it's a leftover. I just added a commit to remove it.